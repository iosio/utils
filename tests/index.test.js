import React from 'react'
import renderer from 'react-test-renderer';

import Component from '../src'

describe('Component', () => {

    it('renders without crashing', () => {
        const tree = renderer.create(
            <Component/>
        );

        expect(tree).toMatchSnapshot();
    })
});

