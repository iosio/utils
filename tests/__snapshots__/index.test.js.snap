// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders without crashing 1`] = `
<div>
  <h2>
    Welcome to React components
  </h2>
</div>
`;
